app/assets/stylesheets/_times-dates-list.scss

```diff
@@ -36,3 +36,7 @@
     }
   }
 }
+
+p.abstract--dates {
+  @extend h3;
```

Comment:
Being picky, but I would recommend avoiding `@extend` as we found it created bugs in the UI Kit. It might be best to simply copy/paste the scss for h3. 
CC: @klepas 

https://www.sitepoint.com/avoid-sass-extend/